lib/rabbit_feed/client.rb

```diff
@@ -160,6 +161,10 @@ def parse_options argv
           opts[:environment] = arg
         end
 
+        o.on '-x', '--prefix VAL', 'Prefix to append to the routing and queue naming allowing exclusive publishing/consuming' do |arg|
```

Comment:
but you have `application` in there, which uses in queue / routing as well. nevertheless, i am looking to remove it